airflow/providers/amazon/aws/hooks/databrew.py

```diff
@@ -0,0 +1,68 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from __future__ import annotations
+
+from airflow.providers.amazon.aws.hooks.base_aws import AwsBaseHook
+
+
+class GlueDataBrewHook(AwsBaseHook):
```

Comment:
If file name is databrew why class is GlueDataBrewHook? Is it intentional?

Reply:
I think the files should be `glue_databrew`. This would align with the boto3 docs and the service docs. I search for 'glue databrew' when looking for info.

Reply:
That's a good catch. I agree, it should be `glue_databrew`

Reply:
@ellisms can you align the classes / file names ?

Reply:
@eladkal yep, will work on this today.